frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts

```diff
@@ -0,0 +1,36 @@
+// import { withResource } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { V2V_VM_IMPORT_TIMEOUT, VM_STATUS } from '../tests/utils/consts';
+import { VirtualMachine } from '../tests/models/virtualMachine';
+import { vmwareVMMultiNicConfig } from './v2v.configs';
+import {
+  // removeLeakedResources,
+  withResource,
+  createResources,
+  deleteResources,
+} from '@console/shared/src/test-utils/utils';
+import { multusNAD } from '../tests/utils/mocks';
+
+describe('Kubevirt create VM using wizard', () => {
+  const leakedResources = new Set<string>();
+
+  beforeAll(async () => {
+    createResources([multusNAD]);
+  });
+
+  afterAll(async () => {
+    deleteResources([multusNAD]);
+  });
+
+  it(
```

Comment:
So this creates only the one VM. You want to loop over the list of VM configs you created.

```
v2vImportConfigs.forEach( (importConfig) => {
  it(
    `Imports VM ${importConfig.name} from VMware Instance`,
    async () => {
      const vm = new VirtualMachine({ name: importConfig.name, namespace: testName });
      await withResource(leakedResources, vm.asResource(), async () => {
        await vm.import(importConfig);
        await vm.waitForStatus(VM_STATUS.Off, V2V_VM_IMPORT_TIMEOUT);
      });
    },
    V2V_VM_IMPORT_TIMEOUT,
  );
})
```

Reply:
I would like not to build a loop that you described, cause basic test should be used as a sanity test, each time when new build is automatically tested. Test with 2 NICs - it is advanced and should be executed in a suite. I don't want them to be executed together